rdr_service/dao/genomics_dao.py

```diff
@@ -3417,19 +3401,9 @@ def get_data_ready_for_w1il_manifest(self, module: str, cvl_id: str):
                 GenomicSetMember.gcManifestSampleSource.ilike('whole blood'),
                 ParticipantSummary.consentForStudyEnrollment == QuestionnaireStatus.SUBMITTED,
 
-                # all data files have been received
-                GenomicGCValidationMetrics.hfVcfReceived == 1,
-                GenomicGCValidationMetrics.hfVcfTbiReceived == 1,
-                GenomicGCValidationMetrics.hfVcfMd5Received == 1,
-                GenomicGCValidationMetrics.cramReceived == 1,
-                GenomicGCValidationMetrics.cramMd5Received == 1,
-                GenomicGCValidationMetrics.craiReceived == 1,
-                GenomicGCValidationMetrics.gvcfReceived == 1,
-                GenomicGCValidationMetrics.gvcfMd5Received == 1,
-
                 ParticipantSummary.consentForGenomicsROR == QuestionnaireStatus.SUBMITTED,
                 GenomicGCValidationMetrics.drcFpConcordance.ilike('pass'),
-                sample_collected_site.siteType != 'diversion pouch',
+                GenomicSetMember.diversionPouchSiteFlag == 0,
```

Comment:
Can we update this to `!=1`, as this is the convention we use in the rest of the queries.

Reply:
Yes, I've committed that change.